pushmanager/static/js/push.js

```diff
@@ -94,18 +95,8 @@ $(function() {
         });
     });
     $('.message-people').live('click', function() {
-        var contents = $(this).siblings('.item-count').text();
-
-        var people_pat = new RegExp(    // person, person (person, person), person (person), person
-            "(?:[a-z]+" +                   // A username, possibly followed by
-                "(?:\\s\\(" +               //   a space and (
-                    "(?:[a-z]+,?\\s?)+" +   //   and one or more usernames (possibly separated by comma and/or a single space)
-                "\\))?" +                   //   followed by a )
-            ",?\\s?" +                      // possibly followed by a command and space
-            ")+", "i")                           // and more of the same
-
-        var people = (people_pat.exec(contents) || [""])[0];
-        PushManager.send_message_dialog(people);
+	var state = $(this).parent().attr('section');
+	PushManager.send_message_dialog(state);
     });
     $('#message-all').live('click', function() {
         var people = PushManager.all_involved_users();
```

Comment:
Need to update & support message-all (connected to the Message All button at the top of a push). It should message everyone in any state in the push (everyone but open requests).